numba/_dispatcher.cpp

```diff
@@ -614,6 +643,20 @@ Dispatcher_call(Dispatcher *self, PyObject *args, PyObject *kws)
     PyThreadState *ts = PyThreadState_Get();
     PyObject *locals = NULL;
 
+    // Check TLS target stack
+    if (use_tls_target_stack) {
+        raii_use_tls_target_stack turn_off(false);
```

Comment:
Why `use_tls_target_stack` is turned off? Does it mean that nested functions will not use target stack?

Reply:
Nested function will not go through this part of the code. The dispatcher is the bridge connecting the Python interpreter code into the compiled code. The change of the target will only need to happen once. 

In this branch, the dispatcher will go into `_call_tls_target` where a new dispatcher will be invoked to compile and execute for the specific target. By turning off the`use_tls_target_stack` here, we stop the new dispatcher from calling `_call_tls_target` unnecessarily.